### PR TITLE
Added channels and updated links

### PR DESCRIPTION
**Pull request by Niyasmkkv**

### CODE REVIEW

File: channels/in.m3u [hunk 1]
```diff
@@ -194,7 +216,7 @@ http://103.199.161.254/Content/jaihind/Live/Channel(Jaihind)/index.m3u8
 #EXTINF:-1 tvg-id="JanTV.in" tvg-name="Jan TV" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="",Jan TV (360p)
 http://jantvstream.in:1935/edge1/sc1jantv.stream_aac/playlist.m3u8
 #EXTINF:-1 tvg-id="JanamTV.in" tvg-name="Janam TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/LWUpmDm.png" group-title="News",Janam TV (720p)
-https://vidcdn.vidgyor.com/janamtv-origin/liveabr/playlist.m3u8
```
Comment (Dum4G) [inline channels/in.m3u:197]: We don't remove alternate sources if those work well

File: channels/in.m3u [hunk 2]
```diff
@@ -454,8 +476,8 @@ https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtu
 https://satsangtv.sanskargroup.in/satsangtvlive.m3u8
 #EXTINF:-1 tvg-id="ServeurTV.in" tvg-name="Serveur TV" tvg-country="IN" tvg-language="English" tvg-logo="" group-title="",Serveur TV (720p)
 https://a.jsrdn.com/broadcast/060753d37e/+0000/c.m3u8
-#EXTINF:-1 tvg-id="ShekinahTV.in" tvg-name="Shekinah TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/6hTshsG.png" group-title="",Shekinah TV [Offline]
-https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCHtYUfPnQci6GoUpPlWfrOg/live
```
Comment (Dum4G) [inline channels/in.m3u:458]: We don't remove alternate sources if those work well

File: channels/in.m3u [hunk 3]
```diff
@@ -506,9 +528,9 @@ https://tna5.bozztv.com/streamlive/index.m3u8
 #EXTINF:-1 tvg-id="SteelbirdMusic.in" tvg-name="Steelbird Music" tvg-country="IN" tvg-language="" tvg-logo="https://i.imgur.com/5r4gCMh.jpg" group-title="Music",Steelbird Music (720p) [Not 24/7]
 http://cdn25.live247stream.com/steelbirdmusic/tv/playlist.m3u8
 #EXTINF:-1 tvg-id="StudioOnePlus.in" tvg-name="Studio One Plus" tvg-country="IN" tvg-language="Telugu" tvg-logo="https://i.imgur.com/DBbajAs.jpg" group-title="Movies",Studio One Plus
-https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCEoaVUxpFlmCxwRBy9yhb1Q/live
```
Comment (Dum4G) [inline channels/in.m3u:509]: We don't remove alternate sources if those work well

File: channels/in.m3u [hunk 4]
```diff
@@ -506,9 +528,9 @@ https://tna5.bozztv.com/streamlive/index.m3u8
...
+http://14.199.164.20:4001/play/a0oh/index.m3u8
 #EXTINF:-1 tvg-id="StudioOneYuva.in" tvg-name="Studio One Yuva" tvg-country="IN" tvg-language="Telugu" tvg-logo="https://i.imgur.com/nACKF3x.jpg" group-title="Movies",Studio One Yuva
-https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCqR2tCs8ufgAjRd6rn4a2wg/live
```
Comment (Dum4G) [inline channels/in.m3u:511]: We don't remove alternate sources if those work well

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/in.m3u (hunk 1 above)
Comment (Niyasmkkv) [reply channels/in.m3u:197]: Those links were taking time 
, That's why updated the new links. Can you fix it now?

Comment (Dum4G) [reply channels/in.m3u:197]: You can add them back into your file, just add them as another channel with the same description

Comment (Niyasmkkv) [reply channels/in.m3u:197]: Added those links as new channels.

Comment (Dum4G) [reply channels/in.m3u:197]: Looks like you have forgot to replace links to the old ones where it's needed

Comment (Niyasmkkv) [reply channels/in.m3u:197]: Oh damn. how do I fix it now?

Comment (Niyasmkkv) [reply channels/in.m3u:197]: Fixed it.But PR test failed again.

File: channels/in.m3u [hunk 5]
```diff
@@ -141,6 +153,10 @@ http://cdn7.live247stream.com/desibeats/tv/playlist.m3u8
 https://live.wmncdn.net/desichannel/7e2dd0aed46b70a5c77f4affdb702e4b.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="DesiPlus.in" tvg-name="Desi Plus" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="",Desi Plus (720p)
 http://cdn2.live247stream.com/desiplus/tv/playlist.m3u8
+#EXTINF:-1 tvg-id="Dhinchaak.in" tvg-name="Dhinchaak" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/T2c6PjU.jpg" group-title="Movies",Dhinchaak
```
Comment (Dum4G) [inline channels/in.m3u:156]:
```suggestion
#EXTINF:-1 tvg-id="Dhinchaak.in" tvg-name="Dhinchaak" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/T2c6PjU.jpg" group-title="Movies",Dhinchaak
```

File: channels/in.m3u (hunk 1 above)
Comment (Dum4G) [reply channels/in.m3u:197]: there you go. You can check linter logs to see where there trailing space or the wrong tag is

**Review by Dum4G — APPROVED** [review]

**Pull request merged**